hwilib/ui/getxpubdialog.ui

```diff
@@ -6,109 +6,97 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>1205</width>
-    <height>158</height>
+    <width>430</width>
+    <height>140</height>
```

Comment:
xpubs are pretty long, so shrinking this makes the xpub display kind of unergonomic.